tools/cellpose/cellpose.xml

```diff
@@ -0,0 +1,154 @@
+<tool id="cellpose" name="Run Cellpose segmentation" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.2">
+    <description>a generalist algorithm for cell and nucleus segmentation</description>
+    <macros>
+	<token name="@TOOL_VERSION@">3.0.1</token>
+	<token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+	    <container type="docker">biocontainers/cellpose:3.0.1_cv1</container>
+	    <requirement type="package" version="0.22.0">scikit-image</requirement>
```

Comment:
either container of packages, not both ...

Reply:
removed the conda